### ISSUE TITLE
Add instructions to run the app

### ISSUE THREAD

**Issue by knjk04** [knjk04/COVID-19-dashboard#1]
Add instructions to run the app in the README in the setup section after the routes have been fixed

**Comment by knjk04** [comment]
Blocked by issue #2

**Comment by knjk04** [comment]
Update: this is still blocked until the default mapping is set.

**Comment by knjk04** [comment]
See commit [383814a](https://github.com/knjk04/COVID-19-dashboard/commit/6d5dd0489fc5eee596469388da86946a281cfce6).

The default mapping still needs to be set, but this will be made into a new issue.
